abstracts/variables.css

```
@@ -66,6 +66,9 @@
   --space-xl: 2.5rem;
   --space-xxl: 4rem;
 
+  /* Border Radii */
+  --border-radius-full: 50%;
```

Consider renaming this variable. May be you can use something like
```suggestion
  --border-radius-circle: 50%;
```